libs/components/forms/src/lib/modules/input-box/input-box.component.ts

```diff
@@ -20,10 +20,12 @@ import { SkyInputBoxHostService } from './input-box-host.service';
 import { SkyInputBoxPopulateArgs } from './input-box-populate-args';
 
 /**
- * Creates a wrapper to provide styling for `input` and `label` elements. To render the
- * component correctly, include the `sky-form-control` CSS class on the `input` element
- * and the `sky-control-label` CSS class on the `label` element. To display a help button beside the label, include a help button element, such as `sky-help-inline`, in the `sky-input-box` element and a `sky-control-help` CSS class on that help button element.
- * To display an error message under the input, include a `sky-status-indicator` component with a `sky-control-error` class in the `sky-input-box` element, and use `*ngIf` to determine whether to display the error message based on the input value.
+ * Creates a wrapper to provide styling for SKY UX input components and  `input`, `textarea`, `select`, and `label` elements.
+ * To render the component correctly, include the `sky-form-control` class on the input element and the `sky-control-label`
+ * class on the `label` element. To display a help button beside the label, include a `sky-help-inline` component with a
```

Comment:
I'll ping Adam about this--I removed it because I thought it was odd we would want to encourage them to use a different button, but maybe there is a use case I'm missing.

Reply:
@blackbaud-johnly Adam confirmed that if a consumer is using a help button with an input it should be a `sky-help-inline` button